build/CopyFilesToStagingDir.ps1

```diff
@@ -172,6 +234,8 @@ PublishFile $FullBuildOutput\WindowsAppRuntime_BootstrapDLL\Microsoft.WindowsApp
 # Tools
 PublishFile $FullBuildOutput\WindowsAppRuntime_MSIXInstallFromPath\WindowsAppRuntime_MSIXInstallFromPath.exe $NugetDir\tools\$Platform
 PublishFile $FullBuildOutput\WindowsAppRuntime_MSIXInstallFromPath\WindowsAppRuntime_MSIXInstallFromPath.pdb $NugetDir\tools\$Platform
+PublishFile $FullBuildOutput\MakeMSIX\MakeMSIX.exe $NugetDir\tools\$Platform
+PublishFile $FullBuildOutput\MakeMSIX\MakeMSIX.pdb $NugetDir\tools\$Platform
```

Comment:
This should make its way into the Main or (more likely) Singleton package and have an AppExecutionAlias. How do you define one to make its way into the TransportPackage and be picked up appropriately by the MSIX creation logic in the Aggregator?